packages/toolkit/src/lib/vdp-sdk/model/mutations.ts

```diff
@@ -63,90 +69,31 @@ export type CreateUserArtiVCModelPayload = {
 };
 
 export type CreateUserModelResponse = {
-  operation: Operation;
+  model: Model;
 };
 
 export async function createUserModelMutation({
-  userName,
+  entityName,
+  isOrg,
   payload,
   accessToken,
 }: {
-  userName: string;
+  entityName: string;
+  isOrg: boolean;
   payload: CreateUserModelPayload;
   accessToken: Nullable<string>;
 }) {
-  const client = createInstillAxiosClient(accessToken, true);
-  if (payload.type === "Local") {
-    try {
-      const formData = new FormData();
-      formData.append("id", payload.id);
-      formData.append("model_definition", payload.model_definition);
-      formData.append("content", payload.configuration.content);
-
-      if (payload.description) {
-        formData.append("description", payload.description);
-      }
-
-      const { data } = await client.post<CreateUserModelResponse>(
-        `${userName}/models/multipart`,
-        formData,
-        {
-          headers: {
-            "Content-Type": "multipart/form-data",
-          },
-        }
-      );
-      return Promise.resolve(data.operation);
-    } catch (err) {
-      return Promise.reject(err);
-    }
-  } else {
-    let input: Record<string, any> = {};
-
-    if (payload.type === "GitHub") {
-      input = {
-        id: payload.id,
-        model_definition: payload.model_definition,
-        description: payload.description,
-        configuration: {
-          repository: payload.configuration.repository,
-          tag: payload.configuration.tag,
-        },
-      };
-    } else if (payload.type === "ArtiVC") {
-      input = {
-        id: payload.id,
-        model_definition: payload.model_definition,
-        description: payload.description,
-        configuration: {
-          url: payload.configuration.url,
-          credential: payload.configuration.credential
-            ? JSON.parse(payload.configuration.credential)
-            : undefined,
-          tag: payload.configuration.tag,
-        },
-      };
-    } else {
-      input = {
-        id: payload.id,
-        model_definition: payload.model_definition,
-        description: payload.description,
-        configuration: {
-          repo_id: payload.configuration.repo_id,
-        },
-      };
-    }
-
-    try {
-      const { data } = await client.post<CreateUserModelResponse>(
-        `${userName}/models`,
-        input
-      );
-
-      return Promise.resolve(data.operation);
-    } catch (err) {
-      return Promise.reject(err);
-    }
+  try {
+    const client = createInstillAxiosClient(accessToken, true);
+
+    const { data } = await client.post<CreateUserModelResponse>(
+      `/${isOrg ? "organizations" : "users"}/${entityName}/models`,
+      payload
+    );
+
+    return Promise.resolve(data.model);
+  } catch (err) {
+    return Promise.reject(err);
```

Comment:
@orangecoloured I think you might confused around entityName and entity.id

Let's take my account for example, its entity name will be `users/summerbud` and it entity id will be `summerbud`

If it's a organization then its entity name might be `organizations/instill-ai` and its entity id is `instill-ai`

I think we can follow how we write the sdk in other services, instead of exposing flag like isOrg, we only need to expose entityName

Reply:
Done